README.md

```diff
@@ -0,0 +1,32 @@
+
+LSST webcommon
+==============
+
+This project is intended to act as a repository for common code which 
+is shared across web applications, both server-side and client-side.
```

Comment:
Client-side and server-side could have very different set of dependencies, from that perspective it may be better to separate them. Pure client package probably would not want dependency (even indirect) on flask.